tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -375,7 +459,19 @@ def test_invalid_routing_plan(nhsd_apim_proxy_url, correlation_id):
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_invalid_message_batch_reference(nhsd_apim_proxy_url, correlation_id):
     """
-    .. py:function:: Test invalid message batch reference
+    .. py:function:: Scenario: A user submitting a request with an invalid \
+        message batch reference receives a 400 'Invalid Value' response
+
```

Comment:
Explain it needs to be a UUID - link to wikipedia page on UUID

Reply:
Addressed in uuid.rst, linked below given/when/then